packages/velo-external-db-core/lib/service/schema.spec.js

```diff
@@ -50,6 +51,15 @@ describe('Schema Service', () => {
         await expect(env.schemaService.removeColumn(ctx.collectionName, ctx.column.name)).resolves.toEqual({})
     })
 
+    test('collections with _id column will have read-write capabilities', async() => {
+        driver.givenListWithIdColumnResult(ctx.dbs)
+        driver.givenSupportedOperations(ctx.schemaOperations)
+
+        const actual = await env.schemaService.list()
+
+        expect( actual ).toEqual({ schemas: readWriteSchemaList(ctx.dbs, ctx.schemaOperations) })
```

Comment:
also here we can have a nicer more readable matcher.
btw, we can drop the await list and pack it into the expect